docs/source/conventions.rst

```diff
@@ -0,0 +1,239 @@
+****************************************
+Conventions
+****************************************
+
+States and Bases
+####################################
+
+Bases
+*******
+Essentially, a basis refers to a set of two eigenstates. The transition between
```

Comment:
Nit: Just wondering if the "Essentially" is needed "A basis refers to a set of two eigenstates. The transition between" or "A basis essentially refers to a set of two eigenstates. The transition between"

Reply:
I agree with you, I'll remove it